src/types/model/activities/tasks/user_task.ts

```diff
@@ -0,0 +1,24 @@
+import {BpmnType} from '../../../constants';
+import {FormField} from '../../types';
+import {Task} from './task';
+
+/**
+ * Describes a BPMN UserTask.
+ *
+ * A user task will hold the current ProcessExecution and wait for user input
```

Comment:
I think `ProcessExecution` is no name here.

```suggestion
 * A user task will hold the current execution of the process and wait for user input
```